mbed_flasher/common.py

```diff
@@ -15,11 +15,16 @@
 """
 
 import logging
+import os
 import time
 import six
 
+from mbed_flasher.return_codes import EXIT_CODE_FILE_MISSING
+from mbed_flasher.return_codes import EXIT_CODE_DAPLINK_USER_ERROR
+
 
 DEFAULT_RETRY_AMOUNT = 3
+ALLOWED_FILE_EXTENSIONS = (".bin", ".hex", ".act", ".cfg")
```

Comment:
.elf file is missing. Or have you checked from client team (I believe) if they are still using those?

Reply:
elf file is not supported for Daplink - based on Russ comments.

Reply:
double check with local K64F by drag-n-drop elf file - Daplink doesn't do anything with that -> not supported like @c1728p9 mentioned.

Reply:
Ok, perhaps they don't use flasher with that then.

Reply:
They can't use daplink with .elf, since daplink is the one not supporting it. Pyocd supports .elf, so maybe that.